Bottom: c425fa5b45de5b4bacb263f4660e37132f8ceb6c
Top:    81652756fab88b67bd124f06942f4137c5dcaff4
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-21 15:13:19 +0000

More work in progress


---

diff --git a/src/polygon1.c b/src/polygon1.c
index 78803a2..189bc25 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -2923,8 +2923,6 @@ M_Set_Parentage (POLYAREA *poly, POLYPARENTAGE parentage)
 
 }
 
-static void clear_marks (POLYAREA * p);
-
 /* just like poly_Boolean but frees the input polys */
 int
 poly_Boolean_free (POLYAREA * ai, POLYAREA * bi, POLYAREA ** res, int action)
